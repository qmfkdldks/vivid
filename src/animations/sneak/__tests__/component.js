import React from "react";
import { shallow } from "enzyme";
import Sneak from "../component";

describe("Sneak", () => {
  it("should render successfully", () => {
    const component = shallow(<Sneak>Hide and sneak</Sneak>);
    expect(component).toMatchSnapshot();
  });
});
