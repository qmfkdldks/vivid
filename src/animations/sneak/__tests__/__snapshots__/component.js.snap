// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sneak should render successfully 1`] = `
<styled.span>
  <Styled(Component)
    transition={
      Object {
        "duration": 3,
      }
    }
    variants={
      Object {
        "start": Object {
          "opacity": Array [
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            0,
          ],
          "rotate": Array [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            10,
            0,
            0,
          ],
          "scaleX": Array [
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1.2,
            0.8,
            1.05,
            1,
            1,
            1,
            1.2,
            0.8,
            1.05,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
          ],
          "scaleY": Array [
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            0.7,
            1.2,
            0.95,
            1,
            1,
            1,
            0.7,
            1.2,
            0.95,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
          ],
          "x": Array [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            -15,
            -15,
            -15,
            -25,
            -25,
            -25,
            -40,
            -40,
            -40,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -50,
            -60,
          ],
          "y": Array [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            -30,
            0,
            -5,
            0,
            0,
            0,
            0,
            -30,
            0,
            -5,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
          ],
        },
        "stop": Object {
          "opacity": 1,
          "rotate": 0,
          "scaleX": 1,
          "scaleY": 1,
          "x": 0,
          "y": 0,
        },
      }
    }
  >
    H
  </Styled(Component)>
  <span>
    ide and sneak
  </span>
</styled.span>
`;
